prometheus.yml

```diff
@@ -0,0 +1,9 @@
+global:
```

Comment:
what is this file being used for? Looks like something we are configuring in our helm-charts later.